package.json

```diff
@@ -17,6 +17,7 @@
   },
   "homepage": "https://github.com/justgage/tachyons-elm#readme",
   "dependencies": {
+    "clean-css": "^4.2.1",
```

Comment:
Maybe this is still a good idea but Tachyons provides a minified version usually. Maybe just downloading that would make this code simpler?

Reply:
Yep, for sure.
Originally I did add the minified tachyons, however I noticed your following comment, so I just minified the tachyons myself : https://github.com/justgage/tachyons-elm/blob/master/scripts/convert.js#L12

```js
const css = fs.readFileSync('./scripts/tachyons.css', 'utf8'); 
// minificated version introduces problems with compound selectors
```
The only compound selectors I noticed were:
`.hide-child .child` : https://github.com/tachyons-css/tachyons/blob/master/css/tachyons.css#L1861
`.nested-copy-line-height`: https://github.com/tachyons-css/tachyons/blob/master/css/tachyons.css#L1939

I've just tested this locally using minified tachyons and they worked fine.
So I'll add the minified tachyons and rename the file to `tachyons.min.css` like Tachyons for clarity.

Reply:
thanks!